CONTRIBUTING.md

```diff
@@ -4,7 +4,9 @@ We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
 Refer to the [Developer Setup](./SETUP.md) document to learn how to prepare your
-environment.
+environment. You can also consult a [GraphQL Guide](./demo/GraphQL.md) which
+replaces the Neo4j components of `SETUP.md`. Optionally, you can also consult
```

Comment:
i think we are going to remove `SETUP.md` so this shouldn't be necessary anymore

Reply:
Added a comment to that end, thinking we would update this when we delete SETUP.md